game/static/game/css/auth.css

```
@@ -12,20 +12,36 @@ html, body {
+    background-color: #0b0b14;
+
+    background-image:
+        radial-gradient(circle at top left,
+        rgba(240,192,64,0.08),
+        transparent 28%),
+
+        radial-gradient(circle at bottom right,
+        rgba(0,200,255,0.08),
+        transparent 30%);
+
+    background-attachment: fixed;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)